packages/core/src/Stores/client-store.js

```diff
@@ -1834,6 +1834,25 @@ export default class ClientStore extends BaseStore {
     }
 
     setAccounts(accounts) {
+        // hide orphan trading account after wallet migration
```

Comment:
Based on this [thread](https://deriv-group.slack.com/archives/C04ST0NNY5B/p1690807082474639) Kirill Sysoev asked to hide trading accounts that are not linked to any wallet, but I'm not sure it's a good idea and I need your advice )) Feel free to delete this code if you have the same doubts as me